conf/openmetadata.yaml

```diff
@@ -394,3 +394,15 @@ web:
   permission-policy:
     enabled: ${WEB_CONF_PERMISSION_POLICY_ENABLED:-false}
     option: ${WEB_CONF_PERMISSION_POLICY_OPTION:-""}
+
+applications:
```

Comment:
we need to move this out of here and into a separate config.
The bundle itself needs to know about its config path.
cc @pmbrull @mohityadav766